src/common/Common.h

```diff
@@ -42,16 +42,14 @@
 
 #define MAX_NETCLIENT_PACKET_SIZE (32767 - 1)               // Client hardcap: int16 with trailing zero space otherwise crash on memory free
 
-enum TimeConstants
-{
-    MINUTE          = 60,
-    HOUR            = MINUTE * 60,
-    DAY             = HOUR * 24,
-    WEEK            = DAY * 7,
-    MONTH           = DAY * 30,
-    YEAR            = MONTH * 12,
-    IN_MILLISECONDS = 1000
-};
+// TimeConstants
+#define MINUTE 60
+#define HOUR MINUTE * 60
+#define DAY HOUR * 24
+#define WEEK DAY * 7
+#define MONTH DAY * 30
+#define YEAR MONTH * 12
+#define IN_MILLISECONDS 1000
```

Comment:
Isnt it possible to do 
```cpp
enum TimeConstants : uint32
```

Reply:
Maybe not with enum I cant remember, but it is possible to do with enum class. Perhaps that avoids the error and lets us avoid #define

Reply:
I've tried that, still got:

```
Arithmetic between floating-point type 'double' and enumeration type 'TimeConstants' is deprecated
```

when multiplying with decimal values

Reply:
hm.. Good enough I guess until we switch to std::chrono